instrumentation/opentelemetry-instrumentation-httpx/src/opentelemetry/instrumentation/httpx/__init__.py

```diff
@@ -330,6 +353,22 @@ def _extract_parameters(
     return method, url, headers, stream, extensions
 
 
+def _normalize_url(
```

Comment:
Please could we have test coverage for this helper and some url format variations

Reply:
The url format depends on the httpx version and so this is already exercised by the current tests in both branches, i.e. I added the helper in order to have the code / tests to work. I would avoid to add a test for an internal helper.